plugins/inventory/iocage.py

```diff
@@ -357,6 +373,21 @@ def get_inventory(self, path):
 
                 results['_meta']['hostvars'][hostname]['iocage_hooks'] = iocage_hooks
 
+        # Optionally, get the jails names from the properties notes.
+        # Requires the notes format "t1=v1 t2=v2 ..."
+        if inventory_hostname_tag:
+            if not get_properties:
+                raise AnsibleError('Jail properties are needed. Enable get_properties')
+            update = {}
+            for hostname, host_vars in results['_meta']['hostvars'].items():
+                tags = dict([tag.split('=') for tag in host_vars['iocage_properties']['notes'].split()])
```

Comment:
I don't think the list literal is needed here, `dict()` should also accept a generator. Also, you should use `tag.split('=', 1)` to avoid errors when you encounter things like `a=b=c`. Finally, I would skip all tags that do not contain `=` (not sure whether that's possible, but if that's possible this also results in errors):
```suggestion
                tags = dict(tag.split('=', 1) for tag in host_vars['iocage_properties']['notes'].split() if '=' in tag)
```